source/common/router/config_impl.cc

```diff
@@ -212,43 +212,6 @@ HashPolicyImpl::generateHash(const Network::Address::Instance* downstream_addr,
   return hash;
 }
 
-std::vector<MetadataMatchCriterionConstSharedPtr>
```

Comment:
I think the extension policy says these source/common changes should go in a separate PR for ease of review.

Reply:
https://github.com/envoyproxy/envoy/pull/4406